components/scream/src/physics/shoc/shoc_functions_f90.cpp

```diff
@@ -50,5 +53,64 @@ void shoc_subroutine(SHOCSubroutineData& d) // example wrapper function
   // shoc_subroutine_c(d.in1, d.in2, d.in3, &d.out1, &d.out2, &d.out3);
 }
 
+SHOCGridData::SHOCGridData(Int shcol_, Int nlev_, Int nlevi_):
+shcol(shcol_), nlev(nlev_), nlevi(nlevi_),
```

Comment:
Strange indenting scheme. Usually class member initialization has 1 level of indentation.